activerecord/lib/active_record/railties/databases.rake

```diff
@@ -145,7 +145,7 @@ db_namespace = namespace :db do
     db_namespace["_dump"].invoke
   end
 
-  # desc 'Pushes the schema to the next version (specify steps w/ STEP=n).'
+  desc "Pushes the schema to the next version (specify steps w/ STEP=n)."
```

Comment:
We explicitly removed the description of this rake task and we still have the same reason to do so. Please keep it hidden. It is not common enough. Users usually use `db:migrate` not `db:forward`